pkg/process/mdoule.go

```diff
@@ -72,3 +73,7 @@ func (m *Module) NotifyStartSuccess() {
 func (m *Module) Shutdown(ctx context.Context, mgr *module.Manager) error {
 	return m.manager.Shutdown()
 }
+
```

Comment:
Can you also rename this file? And check whether there are more similar files with wrong name?